source/extensions/filters/http/buffer/buffer_filter.h

```diff
@@ -11,6 +11,7 @@
 #include "envoy/stats/stats_macros.h"
 
 #include "common/buffer/buffer_impl.h"
+#include "common/common/fmt.h"
```

Comment:
Why added?  Without a reference in this file, it strikes me as likely the include belongs as a narrower scope.

Reply:
oh oops this shouldn't be here